src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1481,17 +1473,19 @@ featurizers:
           - float
           - double
 
+    # Output type should be changed to matrix<nonstd::optional<T>> and input should support all types
+    # after yaml file support nonstd::optional type

```

Comment:
Add a TODO prefix so that this comment appears in searches for "TODO"

Reply:
Good point. Added TODO prefix.